third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -197,9 +200,14 @@ cl_int (CL_API_CALL *web_clGetGLTextureInfo)(cl_mem, cl_gl_texture_info, size_t,
 #define clEnqueueBarrierWithWaitList web_clEnqueueBarrierWithWaitList
 #define clCreateFromGLTexture web_clCreateFromGLTexture
 
-#if OS(LINUX) || OS(ANDROID)
+#if OS(ANDROID)
 #define MAP_FUNC(fn)  { *(void**)(&fn) = dlsym(handle, #fn); }
 #define MAP_FUNC_OR_BAIL(fn)  { *(void**)(&fn) = dlsym(handle, #fn); if(!fn) return false; }
+#else
+#define MAP_FUNC(fn) ASSERT_NOT_REACHED();
```

Comment:
Isn't this problematic in the sense that `WebCL.cpp` will call `init()`, and on !Android this will result in a lot of calls to `ASSERT_NOT_REACHED()` that will crash Crosswalk if assertions are enabled?